greedybear/settings.py

```diff
@@ -13,11 +13,12 @@
 SECRET_KEY = os.environ.get("DJANGO_SECRET", None) or get_random_secret_key()
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get("DEBUG", False) == "True"
+# DEBUG = os.environ.get("DEBUG", False) == "True"
```

Comment:
there is no need to comment this during development. Use the environment variables in the `env_file`. We cannot just keep these changes until the PR is resolved and then revert them at the end. It is not practical and it is error prone.

This is valid for all the other changes you made in this file.

Reply:
As ik, I had to do this due to #23  thus needed to set it without docker-compose.
Inconvenience regretted

Reply:
The ElasticSearch variable was the only one who needed a one line change to have it work properly. The others do not need any change. We can just add a conditional statement for that variable as mentioned in the issue you linked.